src/types.ts

```diff
@@ -152,7 +152,7 @@ export type UTC_TIMESTAMP = string;
 export type AppAuthentication = {
   type: APP_TYPE;
   token: JWT;
-  appId: number;
+  appId: number | string;
```

Comment:
Yep, this would imply a change in `universal-github-app-jwt` first. Do you want me to open an issue there first?

Reply:
> appId: number | `Iv1.${string}`;

I wouldn't do that, because there is a chance that future Client IDs will start with `lv2....` and I also think there are old apps that still have a client ID without the `lv1` prefix